docs/api-docs/channels/channels-toolkit-reference.md

```diff
@@ -34,44 +31,44 @@ All channels must integrate the following endpoints.
 
 The following endpoints are recommended for most channels.
 
```

Comment:
I will defer to @aglensmith to comment on how the tables are formatted in this article.